website/docs/foundations/elevation/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,4 @@
+## Anatomy
```

Comment:
this didn't seem like anatomy but wasn't sure what else to call it?

Reply:
Elevation types?